src/audits/DuplicateId.js

```diff
@@ -25,22 +26,24 @@ axs.AuditRules.addRule({
     url: 'https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_html_02',
     severity: axs.constants.Severity.SEVERE,
     relevantElementMatcher: function(element) {
-        return axs.browserUtils.matchSelector(element, '[id]');
+        if (element.hasAttribute('id')) {
+            var referrers = axs.utils.getIdReferrers(element);
```

Comment:
This makes sense, but makes the heading confusing. Perhaps "Any ID referred to via an IDREF must be unique in the DOM"?